packages/manager/modules/pci/src/projects/project/notebooks/add/add.controller.js

```diff
@@ -1,8 +1,61 @@
-export default class {
+import get from 'lodash/get';
+import {
+  NOTEBOOK_PRIVACY_SETTINGS,
+  NOTEBOOK_RESOURCES,
+} from './notebook.constants';
+
+export default class NotebookAddController {
+  static convertLabels(labels) {
```

Comment:
Could `constructor` be the first method ? It is what's usually done in most programming language